### PR TITLE
feat: add `configPath` and `silent` options

### PR DESCRIPTION
**Pull request by Atinux**
This will allow `@nuxthub/core` to migrate to `nitro-cloudflare-dev` with the same DX.

I also updated the dependencies and throw a better error when `wrangler` is not installed.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -48,14 +54,16 @@ async function nitroModule(nitro: Nitro) {
...
 
-  consola.box(
-    [
-      "🔥 Cloudflare context bindings enabled for dev server",
-      "",
-      `Config path: ${configPath ? relative(".", configPath) : colorize("yellow", "cannot find `wrangler.toml`")}`,
-      `Persist dir: \`${relative(".", persistDir)}\` ${addedToGitIgnore ? colorize("green", "(added to `.gitignore`)") : ""}`,
-    ].join("\n"),
-  );
+  if (!nitro.options.cloudflareDev?.hideConsoleBox) {
```
Comment (pi0) [inline src/index.ts:57]: i think we can simply call it `silent: true` wdyt?

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ You can configure additional options using `cloudflareDev: { }` in `nitro.config
 - Install the latest LTS version of [Node.js](https://nodejs.org/en/)
 - Enable [Corepack](https://github.com/nodejs/corepack) using `corepack enable`
 - Install dependencies using `pnpm install`
-- Build in stub mode using `pnpm build --stub`
+- Prepare the build in stub mode using `pnpm dev:prepare`
```
Comment (pi0) [inline README.md:52]: Please, i rather to make dev experience aligned with `unbuild` command.

File: src/index.ts (hunk 1 above)
Comment (Atinux) [reply src/index.ts]: Renamed

File: README.md (hunk 2 above)
Comment (Atinux) [reply README.md]: As you wish... I still believe the `dev:prepare` command is better in term of DX, people does not need to know what's dark magic is happening under the hood.

Comment (pi0) [reply README.md]: I wish no `dev:prepared` was needed at all  and `build --stub` is not dark magic. At least it is more readable than tracing via another script to find out what it calls.

**Pull request merged**